docker/docker-compose.provider.yml

```diff
@@ -142,6 +157,34 @@ services:
             options:
                 max-size: '100m'
                 max-file: '1'
+    vector:
```

Comment:
We'll need to see how staging copes with an additional container in terms of RAM / CPU when this is pushed to staging.